### ISSUE TITLE
Browser freezes on big chart response

### ISSUE THREAD

**Issue by mhkarimi1383** [louislam/uptime-kuma#3367]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

When loading big responses while getting chart information from backend browser freezes until render completes

### 👟 Reproduction steps

just open an old monitor and select `1w` for chart time

### 👀 Expected behavior

I think loading data and rendering chart in chucks in the better way

### 😓 Actual Behavior

Loading entire data at ones

### 🐻 Uptime-Kuma Version

1.21.3

### 💻 Operating System and Arch

Debian (with Docker)

### 🌐 Browser

Firefox

**Comment by louislam** [comment]
Yes, as there are too many data and we don't have aggregate data yet.

It can only be fixed when [this pr](https://github.com/louislam/uptime-kuma/pull/2750) is finished.

**Comment by louislam** [comment]
Fixed by https://github.com/louislam/uptime-kuma/pull/2750
